src/CMakeLists.txt

```diff
@@ -201,7 +201,12 @@ find_package(Threads REQUIRED)
 target_link_libraries(rdkafka PUBLIC Threads::Threads)
 
 if(WITH_SASL_CYRUS)
-  target_link_libraries(rdkafka PUBLIC ${SASL_LIBRARIES})
+  if (TARGET PkgConfig::SASL)
+    target_link_libraries(rdkafka PUBLIC PkgConfig::SASL)
+  else()
+    target_include_directories(rdkafka PUBLIC ${SASL_INCLUDE_DIRS})
```

Comment:
Just driving by, but why not just add this line? It's just this `IMPORTED_TARGET` thing that requires you to upgrade to CMake 3.6. In fact there are a few missing `target_include_directories` in this CMake build. Perhaps I'll submit a PR.

Reply:
#2641